plugins.mk

```diff
@@ -36,6 +36,17 @@ ebin/$(PROJECT).app:: $(LFE_FILES) | ebin/
 
 endif
 
+LFE_TEST_FILES = $(sort $(call core_find,$(TEST_DIR),*.lfe))
+
+ifneq ($(LFE_TEST_FILES),)
+lfe-test-dir:
+	$(lfe_verbose) PATH=$(PATH):$(DEPS_DIR)/lfe/bin lfec -o test/ $(LFE_TEST_FILES)
```

Comment:
`$(lfe_verbose)` in this case does not work as expected — file names are not listed.

Reply:
Should be `lfe-test-dir: $(LFE_TEST_FILES)`. Then it should print properly.